addons/web/static/src/views/fields/google_slide_viewer/google_slide_viewer.xml

```diff
@@ -4,8 +4,8 @@
     <t t-name="web.GoogleSlideViewer" t-inherit="web.CharField">
         <xpath expr="//t[@t-else='']" position="after">
             <t t-if="url">
-                <div>
-                    <iframe class="o_embed_iframe w-100"
+                <div class="o_google_slide">
+                    <iframe class="o_google_slide_iframe w-100"
```

Comment:
if you don't need the class, do not put it

Reply:
needed to distinguish from `o_field_input_buttons`